delta.js

```diff
@@ -23,9 +23,10 @@ const isLeapYear = (year) => {
 const year2days = (date) => {
   let totalDays = 0;
 
-  for (let year = 0; year <= date.year; year++) {
-    totalDays += isLeapYear(year) ? 366 : 365;
-  }
+  // for (let year = 0; year <= date.year; year++) {
+  //   totalDays += isLeapYear(year) ? 366 : 365;
+  // }
+  totalDays = date.year * 365;
```

Comment:
Are you sure this can be changed like this?

Reply:
Yes, i think so because in monthsForSpecificYear function we already searched for the leap year and returned the Leapyear/nonLeapyear array. And If i consider     totalDays += isLeapYear(year) ? 366 : 365; it adds 1 more day in the end of the leap year so 2 days extra in the leap year one in the February and one in December.

Reply:
And sorry about package-lock.json. I just did npm install and it changed that. I will remove it from this pr.

Reply:
Not sure I understand what you mean. The following test

```js
test("calculates days between year-long period containing leap year", () => {
  const from = { year: 2018, month: 1, day: 2 };
  const to = { year: 2021, month: 2, day: 3 };

  expect(daysBetween(from, to)).toBe(1128);
});
```

will fail with your changes, as it will calculate this to be 1127 days. [See here](https://www.wolframalpha.com/input/?i=days+between+2018-01-02+and++2021-02-03) for what WolframAlpha also says.